Ref/test/int/ref_integration_test.py

```diff
@@ -235,7 +235,8 @@ def test_active_logger_filter(fprime_test_api):
         fprime_test_api.send_and_assert_command("CdhCore.cmdDisp.CMD_NO_OP")
         fprime_test_api.send_and_assert_command("CdhCore.cmdDisp.CMD_NO_OP")
 
-        time.sleep(0.5)
+        # Wait >1s for ComAggregator's 1Hz timeout to flush all downlink
+        time.sleep(1.5)
```

Comment:
This one may not be needed since we're using `send_and_assert_command` but I figured it wouldn't hurt

Reply:
But likely using only `send_and_assert_command` and not `time.sleep()` _would_ work, and would be a better fix... let me know if you'd rather try that